### DISCUSSION TITLE
[Feature] manually add tagged area to photos

### DISCUSSION THREAD

**Discussion by mstewart14** [immich-app/immich#6205] [Feature Request]
### The feature

I'd like to be able to tag individuals whose faces don't appear in the photo (their back is to the camera).  I may want to tag an area of the photo to highlight a particular item or event.  Pets for example or some other item of interest that ML will not be recognizing any time soon.  

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by OverHash** [comment]
Related to https://github.com/immich-app/immich/discussions/2472#discussioncomment-6850486
